homeassistant/components/livisi/__init__.py

```diff
@@ -0,0 +1,57 @@
+"""The Livisi Smart Home integration."""
+from __future__ import annotations
+
+import asyncio
+from typing import Final
+
+from aiolivisi import AioLivisi
+from aionanoleaf import ClientConnectorError
```

Comment:
I don't think we should import from aionanoleaf.

Reply:
You are right, I changed this, thank you!